### PR TITLE
CASMTRIAGE-4359

### PR DESCRIPTION
**Pull request by mitchty**
# Description

Mostly moving a % to an $ so output matches. While here I noticed that a prior merge was missing an updated output function and restored that code. What is here is what was missing from that prior merge.

# Checklist Before Merging

- [ x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: scripts/operations/kubernetes/encryption.sh [hunk 1]
```diff
@@ -281,32 +281,33 @@ restartk8s() {
...
 }
 
-# Need more data about what state the local kube-apiserver processes are in when
-# this fails before taking any action on a real fix.
-debugapiserverstatus() {
-  kubectl get pod --namespace kube-system "kube-apiserver-$(uname -n)"
+# I've been entirely unable to understand/debug why a 5 minute timeout fails,
+# but eventually works for this....
+kubeapiblurb() {
+  printf "check logs for details why via:\nkubectl logs -f --namespace kube-system %s\nkubectl get pod --namespace kube-system %s\nkubectl describe pod --namespace kube-system %s\nyou will need to re-run this command again\n" "${apiserver}" "${apiserver}" "${apiserver}" >&2
```
[the remark this answers is not in the document]
Comment (mitchty) [reply scripts/operations/kubernetes/encryption.sh:310]: %s strings (3x) just works out to:

```$ apiserver=blah  printf "check logs for details why via:\nkubectl logs -f --namespace kube-system %s\nkubectl get pod --namespace kube-system %s\nkubectl describe pod --namespace kube-system %s\nyou will need to re-run this command again\n" "${apiserver}" "${apiserver}" "${apiserver}" >&2

check logs for details why via:
kubectl logs -f --namespace kube-system
kubectl get pod --namespace kube-system
kubectl describe pod --namespace kube-system
you will need to re-run this command again
```

[the remark this answers is not in the document]
Comment (mitchty) [reply scripts/operations/kubernetes/encryption.sh:310]: Its the %s string for each kubectl output basically.

[the remark this answers is not in the document]
Comment (mitchty) [reply scripts/operations/kubernetes/encryption.sh:311]: I could convert it to a here doc if we want

[the remark this answers is not in the document]
Comment (kimjensen-hpe) [reply scripts/operations/kubernetes/encryption.sh]: ah thanks

**Review by kimjensen-hpe — APPROVED** [review]

**Review by bklei — APPROVED** [review]

**Review by rgross727 — APPROVED** [review]

**Review by gbaker-hpe — APPROVED** [review]

**Pull request merged**